addons/sale/views/sale_order_views.xml

```diff
@@ -213,7 +213,7 @@
                     attrs="{'invisible': [('state', 'not in', ['draft'])]}"/>
                 <button name="action_quotation_send" type="object" string="Send PRO-FORMA Invoice" groups="sale.group_proforma_sales" attrs="{'invisible': ['|', ('state', '=', 'draft'), ('invoice_count','&gt;=',1)]}" context="{'proforma': True, 'validate_analytic': True}"/>
                 <button name="action_quotation_send" string="Send by Email" type="object" states="sent,sale" data-hotkey="g" context="{'validate_analytic': True}"/>
-                <button name="action_cancel" type="object" string="Cancel" attrs="{'invisible': ['|', ('state', 'not in', ['draft', 'sent','sale']), ('id', '=', False)]}" data-hotkey="z"/>
+                <button name="action_cancel" type="object" string="Cancel" attrs="{'invisible': [('state', 'not in', ['draft', 'sent','sale'])]}" data-hotkey="z"/>
```

Comment:
What's the point of cancelling an order you didn't even save ? :thinking:

Reply:
I prefer to avoid cosmetic changes in a stable version.
Since it doesn't fix anything, please revert this change.

Reply:
@Feyensv reverted the changes applied for the cancel button